### PR TITLE
Update react-embed and add example using react

### PR DESCRIPTION
**Pull request by lizraeli**
- Update `react-embed` to match the recent updates to `web-embed`
- Take event handlers with an `on*` prefix as props, and map these to events in web-embed. E.g: `onFlowLoaded` -> `FlowLoaded`, `onRedirect` -> `redirect`

### CODE REVIEW

File: packages/react-embed/src/__tests__/EmbedFlow.test.tsx [hunk 1]
```diff
@@ -61,7 +61,7 @@ describe('EmbedFlow component', () => {
         flowLabel="test-flow"
         clientLabel="test-client"
         variantLabel="test-variant"
-        flowloaded={flowloadedMock}
+        FlowLoaded={flowloadedMock}
```
Comment (lizraeli) [inline packages/react-embed/src/__tests__/EmbedFlow.test.tsx:64]: update event handlers to have an `on*` prefix

File: packages/react-embed/README.md [hunk 2]
```diff
@@ -46,10 +46,12 @@ You can add event listeners to flows like `flowloaded`, `redirect` etc. See [all
...
-| flowloaded    | [event listener](https://github.com/formsort/oss/tree/master/packages/web-embed-api#flowloaded---void)                          | no       | `() => { console.log('flow loaded') }`                     |
-| flowclosed    | [event listener](https://github.com/formsort/oss/tree/master/packages/web-embed-api#flowclosed---void)                          | no       | `() => { console.log('flow closed') }`                     |
-| flowfinalized | [event listener](https://github.com/formsort/oss/tree/master/packages/web-embed-api#flowfinalized---void)                       | no       | `() => { console.log('flow finalized') }`                  |
-| redirect      | [event listener](https://github.com/formsort/oss/tree/master/packages/web-embed-api#redirect-url-string--void)                            | no       | `(url: string) => { console.log('redirecting to:', url) }` |
+| Flowloaded    | [event listener](https://github.com/formsort/oss/tree/master/packages/web-embed-api#flowloaded-answers--key-string-any----void)                          | no       | `() => { console.log('flow loaded') }`                     |
```
Comment (fzembow) [inline packages/react-embed/README.md:49]: These should be `on...`

**Review by fzembow — CHANGES_REQUESTED** [review]
don't forget the README

File: packages/react-embed/README.md (hunk 2 above)
Comment (lizraeli) [reply packages/react-embed/README.md]: updated

**Review by fzembow — APPROVED** [review]

**Pull request merged**